enhancements/20230807-rancher-vcluster-addon.md

```diff
@@ -0,0 +1,80 @@
+# Rancher vcluster addon
+
+## Summary
+
+The rancher-vcluster addon leverages [vcluster](https://www.vcluster.com/) to create a vcluster named `rancher-vcluster` in the `rancher-vcluster` namespace.
+
+### Related Issues
+[GH Issue 2679](https://github.com/harvester/harvester/issues/2679)
+
+### Goals
+- Simplify installation and management of an external rancher on an existing harvester cluster.
+
+## Proposal
+
+### User Stories
+
+#### Simplify deployment of rancher for managing harvester
+
+##### User Experience In Detail
+
+Harvester leverages Rancher integration for the following key functionality:
+* fine grained rbac
```

Comment:
The PR lgtm, can we make the first letter of every list upper case for consistency? Thanks.